playground/OpenAIEndToEnd/OpenAIEndToEnd.AppHost/aspire-manifest.json

```diff
@@ -36,12 +36,12 @@
         "http": {
           "scheme": "http",
           "protocol": "tcp",
-          "transport": "http"
+          "transport": "tcp"
```

Comment:
tcp?

Reply:
Currently debugging, behavior seemed random as I had to delete the file to get this change applied, and it seems it should not be tcp.

Reply:
I see the change and the default value. Still can't explain why it didn't get updated when the file existed, or why the other file got a duplicated line until the file was deleted. I switched to a "dev drive", hope this is not a bug.

Reply:
https://github.com/dotnet/aspire/pull/2664